app/questionGroup/get.controller.js

```diff
@@ -81,10 +87,19 @@ const compileInlineConditionalQuestions = questions => {
     const currentQuestion = question
     currentQuestion.answerSchemas = question.answerSchemas.map(schema => {
       if (schema.conditional) {
+        let thisError
+        const errorString = errors[`id-${conditionalQuestions[schema.conditional].questionId}`]
+          ? errors[`id-${conditionalQuestions[schema.conditional].questionId}`].text
+          : null
```

Comment:
What version of Node do we deploy to? If it's 14, we can use the fancy new optional chaining here 

`const errorString = errors[`id-${conditionalQuestions[schema.conditional].questionId}`]?.text`